src/components/CleanHtml.tsx

```diff
@@ -0,0 +1,8 @@
+import dompurify from 'dompurify';
+
+const CleanHtml = ({ dirtyHtml }: { dirtyHtml: string }): JSX.Element => {
+  const cleanHtml = dompurify.sanitize(dirtyHtml);
+  return <div dangerouslySetInnerHTML={{ __html: cleanHtml }} />;
```

Comment:
I think the easiest way to use the "component" property here would be to replace the `div` with a MUI `Box` and pass `component` to it (if that's possible?).